ament_cmake_test/cmake/ament_add_test.cmake

```diff
@@ -30,11 +30,11 @@
 # :param TIMEOUT: the test timeout in seconds, default: 60
 # :type TIMEOUT: integer
 # :param WORKING_DIRECTORY: the working directory for invoking the
-#   command in, default: CMAKE_SOURCE_DIR
+#   command in, default: PROJECT_SOURCE_DIR
```

Comment:
To match the CMake API I think the working directory should be the current source directory (which is `CMAKE_CURRENT_SOURCE_DIR`).

Reply:
Will test. I suspect that would break projects.

Reply:
I looked it up - If we want to be consistent with CMake, it should be `CMAKE_CURRENT_BINARY_DIR` instead. I've made that change. Can you kick off CI to see how badly this breaks things?

https://cmake.org/cmake/help/latest/command/add_test.html
https://cmake.org/cmake/help/latest/prop_test/WORKING_DIRECTORY.html